app/src/main/java/com/github/giommok/softwaredevproject/FirebaseAccount.java

```diff
@@ -32,6 +33,8 @@
     private static String username = "null";
     private static long score = 0;
     private static FirebaseAccount account = null;
+    /*local list of friends */
+    private static ArrayList<FriendItem> friends = new ArrayList<>();
```

Comment:
A `HashSet` would be a better option since we doesn't need duplicates in the list of Friends.

Reply:
Discussed on Discord.